guides/source/active_support_instrumentation.md

```diff
@@ -280,9 +280,10 @@ INFO. Additional keys may be added by the caller.
 
 ### unpermitted_parameters.action_controller
 
-| Key     | Value            |
-| ------- | ---------------- |
-| `:keys` | Unpermitted keys |
+| Key           | Value                                                                 |
+| ------------- | --------------------------------------------------------------------- |
+| `:key`        | The unpermitted keys                                                  |
```

Comment:
Should this be `:key` or `keys`? Why does it change from before?

Reply:
@hidr0 Good catch! Fixed in fe1cb68a3f3505277fe34d88f3f86334a9415dc9.